src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
+    public int getRandomNumber() {
+        Random randomNumber = new Random();
```

Comment:
Creating a new instance of Random every time the 'getRandomNumber' method is called is not efficient. Consider making 'randomNumber' a private class-level field instead.